jina/drivers/cache.py

```diff
@@ -9,11 +10,11 @@
 
 
 class BaseCacheDriver(BaseIndexDriver):
-    """A driver related to :class:`BaseCache`
-    """
+    """A driver related to :class:`BaseCache`."""
 
     def __init__(self, with_serialization: bool = False, *args, **kwargs):
-        """
+        """Create a new BaseCacheDriver.
+
         :param with_serialization: feed serialized Document to the CacheIndexer
```

Comment:
All params need :type: too

Reply:
They're in the method signature, right? That's just duplication of information

Reply:
And the linter doesn't complain about it